packages/code-du-travail-frontend/src/outils/DureePreavisLicenciement/stepReducer.js

```diff
@@ -0,0 +1,45 @@
+import { StepIntro } from "./steps/Introduction";
+import { StepInfoCCn } from "./steps/InfosCCn";
+import { StepResult } from "./steps/Result";
+import { StepInformations } from "./steps/Informations";
+import { StepStatus } from "./steps/Status";
+
+export const initialSteps = [
+  {
+    component: StepIntro,
+    name: "intro",
+    label: "Introduction"
+  },
+  {
+    component: StepStatus,
+    name: "situation",
+    label: "Situation du salarié"
+  },
+  {
+    component: StepInfoCCn,
+    name: "info_cc",
+    label: "Convention collective",
+    isOptionnal: true
```

Comment:
why there is a isOptionnal, info is mandatory, here ?

Reply:
Yes